### DISCUSSION TITLE
error while using the library

### DISCUSSION THREAD

**Discussion by moulichekkapalli** [shankarpandala/lazypredict#407] [General]
models,predictions = clf.fit(X_train_tfidf, X_test_tfidf, y_train, y_test)
 
 error:AttributeError: select_dtypes not found

**Comment by jakemiller649** [comment]
The input does not handle sparse arrays (which are the output of TFIDF). Pass in `X_train_tfidf.toarray()` and `X_test_tfidf.toarray()`

**Comment by 1400vahid** [comment]
i can not install lazypredic. i have some errors: 
1:
![image](https://user-images.githubusercontent.com/98986404/228833815-3cb26e50-2b65-4b85-a23b-9f526c2423c3.png)


2: 

![image](https://user-images.githubusercontent.com/98986404/228834116-2343884e-e39a-408a-8ad9-1ca595832f3b.png)



i google and found some solutions like installing  openssl  or  others. ..but errors have not solved yet.
what must i do?
thanks
